Tests/swift-sdk-swift-tests/IterableAPITests.swift

```diff
@@ -559,53 +555,35 @@ class IterableAPITests: XCTestCase {
     }
     
     func testGetInAppMessages() {
```

Comment:
We need to use this test for RequestCreator.createGetInAppMessagesRequest so that we are testing the request we are sending properly. 
We are still using the backend call to get in app messages but we are not exposing this as a public function. Does that make sense? I am OK if you create a separate Jira to write a test for that and assign that test for me.

Reply:
We still need to test for all these query params.

Reply:
I've renamed it and created a new ticket for the test you specified to minimize delays on this particular PR. Good suggestion!